streams/catalog/pom.xml

```diff
@@ -53,14 +53,6 @@
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
         </dependency>
-        <dependency>
```

Comment:
Catalog deps may not be shipped to storm runtime so is this required ?. If so, do we need to exclude hive-metastore from streamline-cluster as well? I am not sure if it would affect the functionality of SAM to look up hive tables.

Reply:
Even in webservice module it might make conflict between SR. Need to check dependency tree again. Btw, even it conflicts, maybe we have to ship hive-exec as 'core' classifier instead of just removing it, and exclude Avro there. Will check.